### PR TITLE
fixed style

### PR DESCRIPTION
**Pull request by Tarabarnik**

### CODE REVIEW

File: src/main/java/core/basesyntax/Car.java [hunk 1]
```diff
@@ -58,14 +58,16 @@ public void setWeight(Integer weight) {
...
+        }
         Car car = (Car) o;
-        return Objects.equals(model, car.model) &&
-                Objects.equals(engine, car.engine) &&
-                Objects.equals(color, car.color) &&
-                autoType == car.autoType &&
-                Objects.equals(weight, car.weight);
+        return Objects.equals(model, car.model) && Objects.equals(engine, car.engine)
+                && Objects.equals(color, car.color) && autoType == car.autoType
+                && Objects.equals(weight, car.weight);
```
Comment (roman-mate) [inline src/main/java/core/basesyntax/Car.java:72]: each && should be on the new line.

File: src/main/java/core/basesyntax/Car.java [hunk 2]
```diff
@@ -58,14 +58,16 @@ public void setWeight(Integer weight) {
 
     @Override
     public boolean equals(Object o) {
```
Comment (roman-mate) [inline src/main/java/core/basesyntax/Car.java]: suggestion: o -> other.

File: src/main/java/core/basesyntax/Car.java [hunk 3]
```diff
@@ -58,14 +58,16 @@ public void setWeight(Integer weight) {
...
+            return false;
+        }
         Car car = (Car) o;
-        return Objects.equals(model, car.model) &&
-                Objects.equals(engine, car.engine) &&
-                Objects.equals(color, car.color) &&
-                autoType == car.autoType &&
-                Objects.equals(weight, car.weight);
+        return Objects.equals(model, car.model) && Objects.equals(engine, car.engine)
+                && Objects.equals(color, car.color) && autoType == car.autoType
```
Comment (roman-mate) [inline src/main/java/core/basesyntax/Car.java]: why not Objects.equals for autoType?

File: src/main/java/core/basesyntax/Car.java [hunk 4]
```diff
@@ -75,12 +77,10 @@ public int hashCode() {
...
-                ", engine=" + engine +
-                ", color='" + color + '\'' +
-                ", autoType=" + autoType +
-                ", weight=" + weight +
-                '}';
+        return "Car{" + "model='" + model
+                + '\'' + ", engine=" + engine
+                + ", color='" + color
+                + '\'' + ", autoType=" + autoType
+                + ", weight=" + weight + '}';
```
Comment (roman-mate) [inline src/main/java/core/basesyntax/Car.java]: previous formatting is better IMO

Comment (Tarabarnik) [reply src/main/java/core/basesyntax/Car.java]: '+' should be on a new line due to checkstyle

File: src/main/java/core/basesyntax/Car.java (hunk 3 above)
Comment (Tarabarnik) [reply src/main/java/core/basesyntax/Car.java]: this is enum, and this equals was generated by Idea

**Review by roman-mate — APPROVED** [review]

**Pull request merged**